src/input.ts

```diff
@@ -3,6 +3,11 @@ import * as tc from '@actions/tool-cache';
 
 export type Args = {
     version: string
+    access_key: string
+    secret_key: string
+    default_organization_id: string
+    default_project_id: string
+    args: string
```

Comment:
```suggestion
export interface Args {
    version: string;
    access_key: string;
    secret_key: string;
    default_organization_id: string;
    default_project_id: string;
    args: string;
```
We can use an interface here

Reply:
This is flagged as error by linter